File: Iteration 1/code/CSE3063GRP12/src/features/Advisor/CourseApprovalView.java
package features.Advisor;

import core.models.concretes.CourseEnrollment;
import core.exceptions.UnexpectedInputException;
import java.util.ArrayList;

public class CourseApprovalView {
    public void showPendingCourseEnrollments(ArrayList<CourseEnrollment> courseEnrollmentArrayList){
        for(int i = 1;i<courseEnrollmentArrayList.size();i++) {
            System.out.printf(i+". " + courseEnrollmentArrayList.get(i-1).getStudentId());
        }
    }

    public void showErrorMessage(Exception e){
       new UnexpectedInputException();
    }

    public void showPromptMessage() {
        System.out.print("Please enter your selection: ");
    }
}

File: Iteration3/code/CSE3063GRP12/src/features/advisor/course_approval/CourseApprovalView.py
class CourseApprovalView:
    def showPendingCourseEnrollments(self, courseEnrollmentArrayList):
        print("Pending Course Enrollments (enter q for exit)")
        for i, enrollment in enumerate(courseEnrollmentArrayList, start=1):
            print(f"{i}. {enrollment.getStudentId()}")

    def showErrorMessage(self, e):
        print(f"Error: {e.getMessage()}")

    def showPromptMessage(self):
        print("Please enter your selection: ")
